### DISCUSSION TITLE
Welcome to ph-css Discussions!

### DISCUSSION THREAD

**Discussion by phax** [phax/ph-css#86] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by PedroGuedesDT** [comment]
I'm exploring this library to develop a POC for a possible solution.
Yet, in my tests, when i try to parse a valid CSS file that have a selector like this:

:is(p, a)

It complains that it don't expects the comma. But when i check some documentation, it is expected to support multiple values.

Check this:
![image](https://github.com/user-attachments/assets/c5eee6d3-9cc8-4022-8525-d9f03bfcec01)
[Mozilla documentation](https://developer.mozilla.org/en-US/docs/Web/CSS/:is)

Am i doing somthing wrong, or is something that is not previewed to happen here?

**Reply by phax** [reply]
@PedroGuedesDT please file this as an issue. Unfortunately I cannot transfer a single comment into an issue. Will look into it
